changelog/v1.27.5-patch2/base64url.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+- type: FIX
+  issueLink: https://github.com/solo-io/envoy-gloo/issues/329
+  resolvesIssue: false
+  description: >-
+    Fall back to attempting to decode Base64Url if Base64 decoding fails in the Inja transformation
```

Comment:
i'm a little confused by the language here. where exactly does the "fall back" occur when decoding base64 urls? as far as i can see, we are only adding 2 new functions (base64url_encode/decode), but i don't see where we fall back to using it if failure occurs in the regular base64 encoding/decoding